els/categories/api.py

```diff
@@ -0,0 +1,18 @@
+from categories.models import category
+from rest_framework import viewsets, permissions
+from .serializers import categorieserializer
+
+# category Viewset
+
+
+class categoryViewSet(viewsets.ModelViewSet):
```

Comment:
For class names, use Pascal Case
```suggestion
class CategoryViewSet(viewsets.ModelViewSet):
```